src/main/java/levi9/NotificationService/config/AWSSNSConfig.java

```diff
@@ -0,0 +1,26 @@
+package levi9.NotificationService.config;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.sns.AmazonSNSClient;
+import com.amazonaws.services.sns.AmazonSNSClientBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class AWSSNSConfig {
+    public static final String SECRET_KEY = "gAdDOKTnfO095JM0R7pnGJaNm6SP6VbaGI9KJa60";
```

Comment:
since you added secret in code you should make repository (Github) private

Reply:
solved